sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -995,6 +1021,45 @@ def test_simple_schema_parsing(self):
         fields=[string_field, number_field])
     self.assertEqual(expected_table_schema, table_schema)
```

Comment:
Pls expand this test to convert this schema to a dict and compare with the original dict. Also include nested fields.